FirstProject/new-evaluation.html

```diff
@@ -0,0 +1,536 @@
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                        Testing(unit, E2E) <select>
+                        <option selected disabled>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                        Websockets <select>
+                        <option selected disabled>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                        Promises <select>
+                        <option selected disabled>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                    </li>
+
+                    <li>
+                        DOM Manipulation <select>
+                        <option>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                        Event Handling <select>
+                        <option>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                        AJAX <select>
+                        <option>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                        Tooling <select>
+                        <option>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                        Browser Engines <select>
+                        <option>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                    </li>
+                </ul>
+            </fieldset>
+        </form>
+
+        <form class="language_form" style="text-align:center;">
```

Comment:
I should set an action there. Action specifies the place where the action is made.(for example a PHP file) and a method attribute (GET or POST).